consensus/enclave/impl/src/lib.rs

```diff
@@ -424,6 +453,12 @@ impl ConsensusEnclave for SgxConsensusEnclave {
         encrypted_txs_with_proofs: &[(WellFormedEncryptedTx, Vec<TxOutMembershipProof>)],
         root_element: &TxOutMembershipElement,
     ) -> Result<(Block, BlockContents, BlockSignature)> {
+        let blockchain_config = self.blockchain_config.lock()?;
+        let config = blockchain_config
```

Comment:
It might be worth adding a check that `parent_block.version <= config.block_version`?

Reply:
Yea that seems fine, if we test in multiple places then we probably won't lose the test if lmdb gets replaced or something